tests/support/server.rs

```diff
@@ -23,6 +23,7 @@ pub struct Txn {
     pub read_timeout: Option<Duration>,
     pub response_timeout: Option<Duration>,
     pub write_timeout: Option<Duration>,
+    pub chunk_size: Option<usize>,
```

Comment:
When it blocks is important because the internal state of the decoder will change over time. That is, blocking before any bytes are received is a different code path to blocking after bytes are received.